src/sentry/api/invite_helper.py

```diff
@@ -254,6 +255,12 @@ def accept_invite(self, user: User | None = None) -> RpcOrganizationMember | Non
         )
 
         metrics.incr("organization.invite-accepted", sample_rate=1.0)
+        organization_service.schedule_signal(
```

Comment:
After this lands, I can remove the specialized logic in the organization member updated handler https://github.com/getsentry/sentry/blob/master/src/sentry/receivers/outbox/region.py#L110

This is more like how it originally worked.